src/internal/m365/collection/drive/delta_tree_test.go

```diff
@@ -8,11 +8,49 @@ import (
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 
-	odConsts "github.com/alcionai/corso/src/internal/m365/service/onedrive/consts"
 	"github.com/alcionai/corso/src/internal/tester"
 	"github.com/alcionai/corso/src/pkg/path"
 )
 
+// ---------------------------------------------------------------------------
+// helpers
+// ---------------------------------------------------------------------------
+
+var loc = path.NewElements("root:/foo/bar/baz/qux/fnords/smarf/voi/zumba/bangles/howdyhowdyhowdy")
+
+func treeWithRoot() *folderyMcFolderFace {
```

Comment:
nit: could you please go through this function and the helpers below it and switch it to use API functions to manipulate the tree structure instead of manipulating it directly? That will make it so that these helpers require less maintenance going forward as APIs tend to be more stable than internal implementations

For example, this function could use `SetFolder` to add the root node instead of adding it directly to the tree

Reply:
Ah, I'll put a pin in that and get around to it later.  Will save me some merge conflicts down the chain.